build.proj

```diff
@@ -31,6 +31,9 @@
   <Import Project="$(ToolsDir)CodeCoverage.targets" Condition="Exists('$(ToolsDir)CodeCoverage.targets')" />
   <Import Project="$(ToolsDir)PerfTesting.targets" Condition="Exists('$(ToolsDir)PerfTesting.targets') and '$(Performance)' == 'true'"/>
 
+  <!-- Add a hook for custom setup and/or cleanup tasks -->
+  <Import Project="SetupAndCleanup.targets" Condition="Exists('SetupAndCleanup.targets')" />
```

Comment:
I wonder if we shouldn't make this more generic and call it "build.override.targets" or something like that so it is a little clearer that those are special and intended to be unique per-repo.